web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -221,6 +226,25 @@ export const DailyViewTable = (props: Props): JSX.Element => {
     return 4
   }
 
+  const errorIconTheme = createTheme({
```

Comment:
Nice, I think this is the way we're supposed to override CSS in MaterialUI. It's been a struggle with the previous way we were doing it. When it's scoped to a `ThemeProvider` it only affects the children within it, making style changes much easier to reason about.